packages/backend/src/tokens/interstitialRule.ts

```diff
@@ -43,6 +46,17 @@ export const crossOriginRequestWithoutHeader: InterstitialRule = options => {
   return undefined;
 };
 
+export const isPrimaryInDevAndRedirectsToSatellite: InterstitialRule = options => {
```

Comment:
❓ in what scenario will the interstitial load in a primary instance context and be ready to redirect to the satellite? On a full-page reload an app will either load as primary or satellite and then ClerkJS will handle the rest. I must be losing something.

Reply:
Then scenario is the following: 
1.  a satellite app needs to sync, it will redirect to the primary app.
2. primary app loads, it displays its content, (html will render before clerk.js is loaded).
3. clerk.js loads redirects back to satellite with dev_session.

By throwing an interstitial at step 2, we avoid the flickering and flashing of the content coming from the satellite app.

I hope this example helps :)